packages/aragon-wrapper/src/core/proxy/index.js

```diff
@@ -46,8 +46,15 @@ export default class Proxy {
     if (!this.contract.methods[method]) {
       throw new Error(`No method named ${method} on ${this.address}`)
     }
+    
+    const lastParam = params[params.length - 1] 
 
-    return this.contract.methods[method](...params).call()
+    if (typeof lastParam === 'object' && lastParam !== null) {
+      return this.contract.methods[method](...params.slice(0,-1)).call(lastParam)
+    }
+    else {
+      return this.contract.methods[method](...params).call()
```

Comment:
This `else` isn't necessary, since if the previous condition was true, it'd never reach here :).

What do you think about making this a ternary? I like it because the alignment makes it basically a diff of the two options.

```js
return (typeof lastParam === 'object' && lastParam !== null)
  ? this.contract.methods[method](...params.slice(0,-1)).call(lastParam)
  : this.contract.methods[method](...params).call()
```

Reply:
Oh sorry I didn't understand what you meant the other day. Yeah I have to say it is cleaner that way! :)